packages/dataviz/package.json

```diff
@@ -35,27 +35,34 @@
   },
   "dependencies": {
     "classnames": "^2.2.6",
-    "d3": "^6.3.0",
+    "d3": "^6.5.0",
     "date-fns": "^2.16.1",
-    "recharts": "^1.8.5"
+    "geojson": "^0.5.0",
+    "lodash": "^4.17.20",
+    "recharts": "^1.8.5",
+    "topojson-client": "^3.1.0"
```

Comment:
this dependencies seems dead: https://github.com/topojson/topojson-client

Reply:
As discussed together, the library is stable and with very dependancies. So far, it is the one recommended no other alternatives. If a security issue raised concerning this lib then we'll tackle at the time needed.